### DISCUSSION TITLE
CSS splitting per-page

### DISCUSSION THREAD

**Discussion by ghost** [ElMassimo/iles#160] [Q&A] [answered]
Hi!,

I'm wondering if there is any way to split styles per-page, something like:

- If a style appears on 1 page, it's only loaded for that page.
- If a style appears on multiple pages, then add it to global chunk.

Thanks in advance!

**Comment by ElMassimo** [comment, marked as answer]
Hi Ngo!

I have an [experimental branch](https://github.com/ElMassimo/iles/tree/feature/css-code-split) that can automatically split CSS based on usage.

Haven't had enough time to test and update it, but if you'd like to use it as a starting point, feel free to use it!
